src/Serializer/BlockTypeExtractor.php

```diff
@@ -26,10 +26,10 @@ final class BlockTypeExtractor implements PropertyTypeExtractorInterface
     ];
 
     /**
-     * @param ManagerInterface<PageBlockInterface> $blockManager
+     * @param class-string $blockClass
```

Comment:
We should be able to do @param class-string<BlockInterface> $blockClass

Or something like this no ?